service/src/main/java/uk/gov/hmcts/reform/fpl/service/validators/ApplicationDocumentChecker.java

```diff
@@ -0,0 +1,37 @@
+package uk.gov.hmcts.reform.fpl.service.validators;
+
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.fpl.model.ApplicationDocument;
+import uk.gov.hmcts.reform.fpl.model.CaseData;
+import uk.gov.hmcts.reform.fpl.model.common.Element;
+
+import java.util.List;
+
+import static java.util.Collections.emptyList;
+import static org.springframework.util.ObjectUtils.isEmpty;
+
+@Service
+public class ApplicationDocumentChecker implements EventChecker {
+
+    @Override
+    public List<String> validate(CaseData caseData) {
+        return emptyList();
+    }
+
+    @Override
+    public boolean isStarted(CaseData caseData) {
+        final List<Element<ApplicationDocument>> applicationDocuments = caseData.getDocuments();
+        final String documentsToFollow = caseData.getDocumentsToFollow();
+
+        if (!isEmpty(documentsToFollow) || !isEmpty(applicationDocuments)) {
+            return true;
+        }
+
+        return false;
+    }
+
+    @Override
+    public boolean isCompleted(CaseData caseData) {
+        return false;
```

Comment:
is this intentional? Or because it's not a mandatory section do we just return false?

Reply:
yes this is what all the non mandatory event checkers return in this method